packages/contracts-bedrock/deploy-config/devnetL1-template-2.json

```diff
@@ -0,0 +1,58 @@
+{
+  "l1ChainID": 900,
+  "l2ChainID": 902,
+  "l2BlockTime": 2,
+  "maxSequencerDrift": 300,
+  "sequencerWindowSize": 200,
+  "channelTimeout": 120,
+  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
```

Comment:
I think we should re-key all addresses in devnet 2, to avoid accidental overlap or confusion with the other devnet.

Reply:
I can work on addressing both of the above once L2 genesis refactor work is completed since some minor changes will need to be added to this PR to account for new genesis generation steps